nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-shared/src/main/java/org/apache/nifi/kafka/shared/property/provider/StandardKafkaPropertyProvider.java

```diff
@@ -160,9 +165,17 @@ private void setProperty(final Map<String, Object> properties, final String prop
         }
     }
 
-    private boolean isCustomKerberosLoginFound() {
+    private static boolean isCustomKerberosLoginFound() {
+        return isClassFound(SASL_GSSAPI_CUSTOM_LOGIN_CLASS);
+    }
+
+    public static boolean isIAMCallbackHandlerFound() {
```

Comment:
Recommend adjusting the method name to be more specific:
```suggestion
    public static boolean isAwsMskIAMCallbackHandlerFound() {
```